app/views/public_relations/country_status.html.erb

```diff
@@ -58,7 +58,7 @@
             <td><%= link_to 'Show', public_relation, :class => 'button-input' %></td>
             <td><%= link_to 'Edit', edit_public_relation_path(public_relation), :class => 'button-input' %></td>
             <td><%= link_to 'Destroy', public_relation, method: :delete, :class => 'button-input', data: { confirm: 'Are you sure?' } %></td>
-            <td><%= link_to public_relation.country, country_pr_status_path( public_relation.country) %></td>
+            <td><%= link_to public_relation.team.team_name, country_pr_status_path( public_relation.team.team_name) %></td>
```

Comment:
This happens often enough that we might as well define a `team_name` method on PublicRelation, Income, and BonusCredit to get the team_name through the team.